### PR TITLE
modules/SceAppUtil: fix crash if data is null.

### PR DESCRIPTION
**Pull request by Zangetsu38**
Credit to @frangarcj for found fix.

- Fix first lbp crash on saevdata have null data

### CODE REVIEW

File: vita3k/gui/src/app_selector.cpp [hunk 1]
```diff
@@ -304,7 +304,7 @@ void draw_app_selector(GuiState &gui, HostState &host) {
             ImGui::SetColumnWidth(2, COLUMN_SIZE);
             ImGui::SetColumnWidth(3, COLUMN_SIZE);
         }
-        ImGui::SetWindowFontScale(!gui.live_area_font_data.empty() ? 0.82f * scal.x: 1.f);
+        ImGui::SetWindowFontScale(!gui.live_area_font_data.empty() ? 0.82f * scal.x : 1.f);
         ImGui::PushStyleColor(ImGuiCol_Text, GUI_COLOR_TEXT);
```
Comment (1whatleytay) [inline vita3k/gui/src/app_selector.cpp:308]: Not sure if this is the pr for this...

File: vita3k/modules/SceAppUtil/SceAppUtil.cpp [hunk 2]
```diff
@@ -242,7 +242,7 @@ EXPORT(int, sceAppUtilSaveDataDataSave, SceAppUtilSaveDataFileSlot *slot, SceApp
         }
     }
 
-    if (slot) {
+    if (slot->slotParam) {
```
Comment (1whatleytay) [inline vita3k/modules/SceAppUtil/SceAppUtil.cpp:245]:
```suggestion
    if (slot && slot->slotParam) {
```
if slot can still be nullptr

File: vita3k/gui/src/app_selector.cpp (hunk 1 above)
Comment (Zangetsu38) [reply vita3k/gui/src/app_selector.cpp:308]: ? you have forget, obligatory need run format for pr now :)

File: vita3k/modules/SceAppUtil/SceAppUtil.cpp (hunk 2 above)
Comment (Zangetsu38) [reply vita3k/modules/SceAppUtil/SceAppUtil.cpp:245]: ah i see

File: vita3k/gui/src/app_selector.cpp (hunk 1 above)
Comment (1whatleytay) [reply vita3k/gui/src/app_selector.cpp:308]: Well frick lol

**Comment by Zangetsu38** [comment]
Done

**Pull request merged**